spanner/changestreams/README.md

```diff
@@ -0,0 +1,42 @@
+# Cloud Spanner Change Stream Sample
+
+## Setup
+
+This sample requires [Java](https://www.java.com/en/download/) and [Maven](http://maven.apache.org/) for building the application.
+
+1.  **Follow the set-up instructions in [the documentation](https://cloud.google.com/java/docs/setup).**
+
+2.  Enable APIs for your project.
+    [Click here](https://console.cloud.google.com/flows/enableapi?apiid=spanner.googleapis.com&showconfirmation=true)
+    to visit Cloud Platform Console and enable the Google Cloud Spanner API.
+
+3.  Create a Cloud Spanner instance and database via the Cloud Plaform Console's
+    [Cloud Spanner section](http://console.cloud.google.com/spanner).
+
+4.  Enable application default credentials by running the command `gcloud auth application-default login`.
+
+## Build and Run
+
+Please use the same prefix for the following commands to manage resources properly.
+
+1. Run the following Maven command to build the application:
+    ```
+    mvn clean package
+    ```
+   
+2. Run the following Maven command to:
+   1. Create a table and a change stream.
+   2. Insert test data to the created table and run change stream queries.
+   3. Drop the created table and change stream.
+
+    ```
+    mvn exec:java -Dexec.mainClass=com.example.spanner.changestreams.ChangeStreamSample -Dexec.args="my-instance my-db prefix"
+    ```
+
+## Build and Run
+
+Run the following Maven command to run unit test:
```

Comment:
Is the following a unit test or an integration test?

Reply:
Integration test, updated.